src/_pytest/compat.py

```diff
@@ -91,11 +91,12 @@ def getfuncargnames(function, is_method=False, cls=None):
     be treated as a bound method even though it's not unless, only in
     the case of cls, the function is a static method.
 
+    The name parameter should be the original name in which the function was collected.
```

Comment:
```suggestion
    The name parameter should be the original name of the collected function.
```
?

Reply:
Could mention that the default is `function.__name__` then maybe.. (but I'm -0 myself)